### PR TITLE
add debug script

### PR DESCRIPTION
**Pull request by ErikJiang**
#### What type of PR is this?
/kind enhancement

#### What this PR does / why we need it:
Add a debugging script to facilitate debugging Kubespray Ansible playbooks.

#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: examples/debug/debug.sh [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+apiVersion: kubean.io/v1alpha1
+kind: ClusterOperation
+metadata:
+  name: ${CLUSTER_NAME}-install-ops
+spec:
+  cluster: ${CLUSTER_NAME}
+  image: ghcr.m.daocloud.io/kubean-io/spray-job:${KUBEAN_VERSION}
+  actionType: playbook
+  action: cluster.yml
+  preHook:
```
Comment (tu1h) [inline examples/debug/debug.sh:135]: For Debug's purpose, is there no need to add Prehook and Posthook? `sleep infinity` may  be the main Action

File: examples/debug/debug.sh [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+#
+# 4. delete custom resources.
+# kubectl delete cluster debug
+#
+
+DEBUG_NODE_ADDR="172.168.10.66"
+DEBUG_NODE_USER="root"
+DEBUG_NODE_PASS="1q2w3e4r"
+
+KUBEAN_VERSION="v0.13.4"
```
Comment (tu1h) [inline examples/debug/debug.sh:23]: It should be renamed as `SPRAY_JOB_VERSION`

File: examples/debug/debug.sh [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+  hosts.yml: |
+    all:
+      hosts:
+        node1:
+          ip: ${DEBUG_NODE_ADDR}
+          access_ip: ${DEBUG_NODE_ADDR}
+          ansible_host: ${DEBUG_NODE_ADDR}
+          ansible_connection: ssh
+          ansible_user: ${DEBUG_NODE_USER}
+          ansible_password: ${DEBUG_NODE_PASS}
```
Comment (tu1h) [inline examples/debug/debug.sh:44]: Will `pubkey` login be supported?

File: examples/debug/debug.sh (hunk 1 above)
Comment (ErikJiang) [reply examples/debug/debug.sh:135]: That's correct, the main effective action is `sleep infinity`, but users may also want to know the specific cluster LCM Ansible operation commands by viewing the `entrypoint.sh` script file.

File: examples/debug/debug.sh (hunk 3 above)
Comment (ErikJiang) [reply examples/debug/debug.sh:44]: This script is intended to provide a simple debugging environment only in non-production environments. Using pubkey would involve operations such as key generation and adding secrets, which would complicate the script. It is not currently being considered unless there is a valid reason to justify the necessity.

File: examples/debug/debug.sh (hunk 1 above)
Comment (tu1h) [reply examples/debug/debug.sh:135]: That could be comments on the top of scripts to indicate the usage.

File: examples/debug/debug.sh (hunk 2 above)
Comment (ErikJiang) [reply examples/debug/debug.sh:23]: At the same time, these parameters can be considered to accept external environment variables.

**Pull request merged**